incubator/hnc/e2e/tests/issues_test.go

```diff
@@ -7,27 +7,34 @@ import (
 var _ = Describe("Issues", func() {
 
 	const (
-		nsParent = "issues-parent"
-		nsChild = "issues-child"
+		nsParent = "parent"
+		nsChild = "child"
+		nsSub1 = "sub1"
+		nsSub2 = "sub2"
+		nsSub1Sub1 = "sub1-sub1"
+		nsSub2Sub1 = "sub2-sub1"
+		nsSubSub2 = "sub-sub2"
+		nsSubChild = "sub-child"
 	)
 
 	BeforeEach(func() {
-		cleanupNamespaces(nsParent, nsChild)
+		cleanupNamespaces(nsParent, nsChild, nsSub1, nsSub2, nsSub1Sub1, nsSub2Sub1, nsSubSub2, nsSubChild)
+		mustRun("kubectl get ns")
 	})
 
 	AfterEach(func() {
-		cleanupNamespaces(nsParent, nsChild)
+		cleanupNamespaces(nsParent, nsChild, nsSub1, nsSub2, nsSub1Sub1, nsSub2Sub1, nsSubSub2, nsSubChild)
 	}) 
 
-	It("Should test issue #328: remove obsolete conditions", func() {
+	It("Should remove obsolete conditions - issue #328", func() {
```

Comment:
Nice change! Since we will add new test cases for removing obsolete `SubnamespaceAnchorMissing` condition, maybe change this to say "... obsolete conditions `CannotPropagateObject` and `CannotUpdateObject`..." to be more specific?

Reply:
done